global.pri

```diff
@@ -41,8 +41,6 @@ run-without-python {
     # from <https://docs.python.org/3/c-api/intro.html#include-files>:
     # "Since Python may define some pre-processor definitions which affect the standard headers on some systems, you must include Python.h before any standard headers are included."
     CONFIG += python
-    QMAKE_CFLAGS += -include Python.h
```

Comment:
why?

Reply:
Was giving me build errors to preinclude that file. If the current Natron build can do it without these flags, which I think it can and it is already included in many places of the codebase where it's needed.